src/app/teacher/color-chooser/color-chooser.component.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MatChipsModule } from '@angular/material/chips';
+
+@Component({
+  selector: 'color-chooser',
+  templateUrl: './color-chooser.component.html',
+  styleUrls: ['./color-chooser.component.scss'],
+  standalone: true,
+  imports: [CommonModule, MatChipsModule]
+})
+export class ColorChooserComponent {
+  @Output() chooseColorEvent: EventEmitter<string> = new EventEmitter();
```

Comment:
Does this emit a string or some sort of an event? Is it MatChipSelectionChange? https://material.angular.io/components/chips/api#MatChipEditedEvent